zerver/views/webhooks/hellosign.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from six.moves import range
+from typing import Dict, Any, Optional, Union
+
+def format_body(signatories, model_payload):
+    # type: (List[Dict[str, Any]], Dict[str, Any]) -> Any
+    result = ["The {contract_title}"] # type: Any
+    for i, signatory in enumerate(signatories):
+        if signatory['status_code'] == 'awaiting_signature':
+            result.append(" is awaiting the signature of {name_")
+            result.append("{i}".format(i=i))
+            result.append('}')
+        elif signatory['status_code'] in ['signed', 'declined']:
+            result.append(" was just {status_")
+            result.append("{i}".format(i=i))
+            result.append("}")
+            result.append(" by {name_")
+            result.append("{i}".format(i=i))
+            result.append("}")
```

Comment:
So do this:
```Python
result = ["The {}".format(model_payload['contract_title']]
# For loop
# if awaiting signature
result.append(" is awaiting the signature of {}".format(model_payload['name_1'])
# else 
result.append(" was just {} by {}".format(model_payload['status_1'], model_payload['name_1'])
```
But I think easier would be to keep separate variables like `awaiting_signatures` and `signed_or_declined`.